src/Components/Form_register/Form_Register.js

```diff
@@ -0,0 +1,155 @@
+                    </td>
+                    <td > <input type='password' name="Password" id="Password" onChange={ (e) => this.HandleChangeInput(e) } /> </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="Avatar"> Avatar:</label>
+                    </td>
+                    <td > <input name='Avatar' type="file" id="Avatar" onChange={ (e) => this.HandleChangeInput(e) } /> </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="PhoneNumber"> Số điện thoại:</label>
+                    </td>
+                    <td > <input  name="PhoneNumber" type="number" id="PhoneNumber" onChange={ (e) => this.HandleChangeInput(e) } />
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="Age"> Tuổi:</label>
+                    </td>
+                    <td > <input name="Age" type="number" id="Age" onChange={ (e) => this.HandleChangeInput(e) } />
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="Gender"> Giới tính:</label>
+                    </td>
+                    <td >
+                    <label>
+                    <input name="Gender" type="radio" value="Male" onClick={ (e) => this.onChangeGenger(e.target.value) } /> Male
+                    </label> <br />
+                    <label>
+                    <input name="Gender" type="radio" value="Female" onClick={ (e) => this.onChangeGenger(e.target.value) }  /> Female
+                    </label>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="favorite"> Sở thích:</label>
+                    </td>
+                    <td >
+                    <label >
+                    <input name="favorite" type="checkbox" value="Đá bóng" onClick={ (e) => this.onChangeFavorite(e.target.value) }/> Đá bóng
+                    </label> <br />
+                    <label >
+                    <input name="favorite" type="checkbox" value="Cầu lông" onClick={ (e) => this.onChangeFavorite(e.target.value) } /> Cầu lông
+                    </label><br />
+                    <label >
+                    <input name="favorite" type="checkbox" value='Đua thuyền' onClick={ (e) => this.onChangeFavorite(e.target.value) } /> Đua thuyền
+                    </label>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                    <label htmlFor="Note"> Ghi chú </label>
+                    </td>
+                    <td > <textarea id="Note" name="Note" onChange={ (e) => this.HandleChangeInput(e) } > </textarea>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+            <p  onClick={ () => this.onSubmit() } >Submit</p>
```

Comment:
Nếu sửa thì sửa như thế này

```suggestion
            <p onClick={this.onSubmit} >Submit</p>
```
Nhưng không ai đi click vào thẻ `p` cả,
Thường thì sẽ click vào thẻ `a` hoặc `button`, trong trường hợp này hãy dùng button